src/sentry/api/event_search.py

```diff
@@ -100,78 +74,83 @@ def translate(pat):
 aggregate_rel_date_filter = negation? aggregate_key sep operator? rel_date_format
 
 # has filter for not null type checks
-has_filter           = negation? "has" sep (search_key / search_value)
-is_filter            = negation? "is" sep search_value
-
-aggregate_key        = key open_paren function_arg* closed_paren
-search_key           = key / quoted_key
-search_value         = quoted_value / value
-value                = ~r"[^()\s]*"
-in_value             = ~r"[^(),\s]*[^\],\s)]"
-numeric_value        = ~r"([-]?[0-9\.]+)([kmb])?(?=\s|\)|$|,|])"
-boolean_value        = ~r"(true|1|false|0)(?=\s|\)|$)"i
-quoted_value         = ~r"\"((?:[^\"]|(?<=\\)[\"])*)?\""s
-key                  = ~r"[a-zA-Z0-9_\.-]+"
-function_arg         = space? key? comma? space?
-# only allow colons in quoted keys
-quoted_key           = ~r"\"([a-zA-Z0-9_\.:-]+)\""
-explicit_tag_key     = "tags[" search_key "]"
-text_key             = explicit_tag_key / search_key
-text_value           = quoted_value / in_value
-
+has_filter = negation? "has" sep (search_key / search_value)
+is_filter  = negation? "is" sep search_value
+
+aggregate_key    = key open_paren function_arg* closed_paren
+search_key       = key / quoted_key
+search_value     = quoted_value / value
+value            = ~r"[^()\s]*"
+in_value         = ~r"[^(),\s]*[^\],\s)]"
+numeric_value    = ~r"([-]?[0-9\.]+)([kmb])?(?=\s|\)|$|,|])"
+boolean_value    = ~r"(true|1|false|0)(?=\s|\)|$)"i
+quoted_value     = ~r"\"((?:[^\"]|(?<=\\)[\"])*)?\""s
+key              = ~r"[a-zA-Z0-9_\.-]+"
+function_arg     = ws key? comma? ws
+quoted_key       = ~r"\"([a-zA-Z0-9_\.:-]+)\""
+explicit_tag_key = "tags[" search_key "]"
+text_key         = explicit_tag_key / search_key
+text_value       = quoted_value / in_value
+
+# Format tokens
 date_format          = ~r"\d{4}-\d{2}-\d{2}(T\d{2}:\d{2}:\d{2}(\.\d{1,6})?)?Z?(?=\s|\)|$)"
 alt_date_format      = ~r"\d{4}-\d{2}-\d{2}(T\d{2}:\d{2}:\d{2}(\.\d{1,6})?(\+\d{2}:\d{2})?)?(?=\s|\)|$)"
 rel_date_format      = ~r"[\+\-][0-9]+[wdhm](?=\s|\)|$)"
 duration_format      = ~r"([0-9\.]+)(ms|s|min|m|hr|h|day|d|wk|w)(?=\s|\)|$)"
 percentage_format    = ~r"([0-9\.]+)%"
 
-# NOTE: the order in which these operators are listed matters
-# because for example, if < comes before <= it will match that
-# even if the operator is <=
+# NOTE: the order in which these operators are listed matters because for
+# example, if < comes before <= it will match that even if the operator is <=
+operator             = ">=" / "<=" / ">" / "<" / "=" / "!="
 or_operator          = ~r"OR(?![^\s])"i
 and_operator         = ~r"AND(?![^\s])"i
-operator             = ">=" / "<=" / ">" / "<" / "=" / "!="
 open_paren           = "("
 closed_paren         = ")"
 open_bracket         = "["
 closed_bracket       = ~r"\](?=\s|$)"
 sep                  = ":"
-space                = " "
 negation             = "!"
 comma                = ","
-spaces               = ~r"\ *"
+ws                   = " "*
```

Comment:
Would using the long-form name `whitespace` be helpful?

Reply:
I shortened it up since I think `ws` or even `_` is a pretty common token name in grammars.

Though @wedamija makes a good point that it isn't technically whitespace, just spaces.